docs/modeling/bounding-boxes.rst

```diff
@@ -0,0 +1,216 @@
+.. _bounding-boxes:
+
+Efficient Model Rendering with Bounding Boxes 
+=============================================
+
+.. versionadded:: 1.1
+
+All `astropy.modeling.Model` subclasses have a ``bounding_box`` attribute that 
+can be used to set the limits over which the model is significant. This greatly 
+improves the effciency of evaluation when the input range is much larger than 
+the characteristic width of the model itself. For example, to create a sky model 
+image from a large survey catalog, each source should only be evaluated over the 
+pixels to which it contributes a significant amount of flux. This task can 
+otherwise be computationally prohibitive on an average CPU. 
+
+The `astropy.modeling.render_model` function can be used to evaluate a model on 
+an input array, or coordinates, limiting the evaluation to the ``bounding_box`` 
+region if it is set.
+
+Using the Bounding Box
+-----------------------
+
+``bounding_box`` is a property that returns a tuple of tuples defining the 
+limits for each input. ``((y_low, y_high), (x_low, x_high), (z_low, z_high))``. 
+Note that the ``'y'``-coordinate 
+comes first for ``model.n_inputs > 1`` following the `numpy` array indexing 
+convention.
+
+By default no bounding box is set (``bounding_box`` is `None`), except for 
+individual model subclasses that have a ``bounding_box_default`` function 
+defined. ``bounding_box_default`` returns the minimum rectangular region 
+symmetric about the position that fully contains the model. If a model is not 
+spatially discrete, e.g. ``Gaussian1D``, the relative error is no more than 
+:math:`10^{-7}`.
+
+        >>> from astropy.modeling import models
+        >>> import numpy as np
+
+        >>> g1 = models.Gaussian1D(stddev=3)
+        >>> g1.amplitude = (g1.stddev * np.sqrt(2 * np.pi)) ** -1
+        >>> g1.bounding_box
+        (-16.5, 16.5)
+
+        >>> x1, x2 = g1.bounding_box
+        >>> x = np.linspace(x1, x2, 100)
+        >>> flux = np.trapz(g1(x), x)
+        >>> print 'Total Flux Error = %.2e' % (1 - flux)
+        Total Flux Error = 3.92e-08
+
+The limits can be set manually by passing an array-like object of 
+shape ``(model.n_inputs, 2)`` or ``(2 * model.n_inputs,)``. 
+
+        >>> g2 = models.Gaussian2D(x_stddev=1, y_stddev=2)
+        >>> g2.bounding_box
+        ((-11.0, 11.0), (-5.5, 5.5))
+
+        >>> g2.bounding_box = [-4.5, 4.5, -3.0, 3.0]
+        >>> g2.bounding_box
+        ((-4.5, 4.5), (-3.0, 3.0))
+
+The default is reset with ``'auto'``.
+
+        >>> g2.bounding_box = 'auto'
+        >>> g2.bounding_box
+        ((-11.0, 11.0), (-5.5, 5.5))
+
+The ``bounding_box`` is turned off with `None`.
+
+        >>> g2.bounding_box = None
+        >>> g2.bounding_box
+        >>> 
```

Comment:
Remove the extra `>>>`
Maybe make the previous statement `print(g2.bounding_box)` so that the `None` shows up as output and it's clear to the reader that this is the value stored?